argilla-server/src/argilla_server/apis/errors/v1/exception_handlers.py

```diff
@@ -0,0 +1,27 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from fastapi import FastAPI, status
+from fastapi.responses import JSONResponse
+
+import argilla_server.errors.future as errors
+
+
+def add_exception_handlers(app: FastAPI):
+    @app.exception_handler(errors.UnprocessableEntityError)
+    async def unprocessable_entity_error_exception_handler(request, exc):
+        return JSONResponse(
+            status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
+            content={"code": exc.code, "message": exc.message},
```

Comment:
One suggestion from @frascuchon is the possibility of returning a `content` compatible with the API v0 errors:

```suggestion
            content={
                "detail": {
                    "code": exc.code,
                    "params": {
                        "detail": exc.message
                    },
                },
            },
```

Add a comment as well noticing how we should return to the simple version once we remove API v0.

Reply:
After discussing it with @damianpumar we agreed into left this new more simple way of sending the error. Damián will manage this without any problem and in the future we can improve and remove the old API v0 errors structure.